addons/mail/tests/discuss/test_discuss_channel_access.py

```diff
@@ -423,6 +423,17 @@ def test_10_discuss_channel_member_access(self):
                         f"{user_key, channel_key, membership, target, operation} should raise access error"
                     ) from e
 
+    def test_11_discuss_channel_invitation_from_token(self):
```

Comment:
may be better in odoo/addons/mail/tests/discuss/test_discuss_channel_as_guest.py
(some day we should rename it to test public page or something like that)